src/picture.rs

```diff
@@ -84,58 +84,47 @@ pub(crate) struct pic_ctx_context {
 }
 
 pub unsafe extern "C" fn dav1d_default_picture_alloc(
-    p: *mut Dav1dPicture,
+    p_c: *mut Dav1dPicture,
     cookie: *mut c_void,
 ) -> Dav1dResult {
-    if !(::core::mem::size_of::<Rav1dMemPoolBuffer>() as c_ulong <= 64 as c_ulong) {
-        unreachable!();
-    }
-    let hbd = ((*p).p.bpc > 8) as c_int;
-    let aligned_w = (*p).p.w + 127 & !(127 as c_int);
-    let aligned_h = (*p).p.h + 127 & !(127 as c_int);
-    let has_chroma =
-        ((*p).p.layout as c_uint != Rav1dPixelLayout::I400 as c_int as c_uint) as c_int;
-    let ss_ver = ((*p).p.layout as c_uint == Rav1dPixelLayout::I420 as c_int as c_uint) as c_int;
-    let ss_hor = ((*p).p.layout as c_uint != Rav1dPixelLayout::I444 as c_int as c_uint) as c_int;
-    let mut y_stride: ptrdiff_t = (aligned_w << hbd) as ptrdiff_t;
-    let mut uv_stride: ptrdiff_t = if has_chroma != 0 {
-        y_stride >> ss_hor
-    } else {
-        0
-    };
+    assert!(::core::mem::size_of::<Rav1dMemPoolBuffer>() <= 64);
```

Comment:
`std::mem` is already imported at the top of the file, so could we do just `mem::size_of` here?

Reply:
Yeah, we could, but I may clean this up more and remove these entirely so I didn't bother changing it here.